device-types/Cisco/C9500-12Q.yaml

```diff
@@ -0,0 +1,48 @@
+---
+manufacturer: Cisco
+model: Catalyst 9500-12Q
+slug: cisco-c9500-12q
+part_number: C9500-12q
+u_height: 1
+is_full_depth: true
+comments: '[Cisco Catalyst 9500 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9500-series-switches/nb-06-cat9500-ser-data-sheet-cte-en.html)'
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
+power-ports:
+  - name: PS-0
+    type: iec-60320-c14
+    maximum_draw: 950
+  - name: PS-1
+    type: iec-60320-c14
+    maximum_draw: 950
```

Comment:
Are the PSUs in this unit modular? If so, you should create `module-bays` for the PSUs and then the associated modular PSUs should be added to the module-type folder if they do not already exist

Reply:
done